compiler-rt/test/profile/ContinuousSyncMode/image-with-no-counters.c

```diff
@@ -1,8 +1,8 @@
-// REQUIRES: darwin
+// REQUIRES: target={{.*(darwin|aix).*}}
 
 // RUN: echo "static void dead_code(void) {}" > %t.dso.c
-// RUN: %clang_profgen -fcoverage-mapping -O3 -dynamiclib -o %t.dso.dylib %t.dso.c
-// RUN: %clang_profgen -fcoverage-mapping -O3 -o %t.exe %s %t.dso.dylib
+// RUN: %clang_profgen_cont -fcoverage-mapping -O3 %shared_lib_flag -o %t.dso.dylib %t.dso.c
+// RUN: %clang_profgen_cont -fcoverage-mapping -O3 -o %t.exe %s %t.dso.dylib
```

Comment:
I tried enabling this on Linux but couldn't. As far as I investigated, looks like bias var has a few issues. Does this really work on AIX?

Reply:
it passes on AIX according to the latest [buildbot](https://lab.llvm.org/buildbot/#/builders/64/builds/1806), (check-all [output](https://lab.llvm.org/buildbot/api/v2/logs/9181507/raw))

Reply:
Thanks. Let me know if you met any issues re. bias var.

Reply:
what problem are you encountering on linux with this testcase?

Reply:
`bias` is not emitted if no instrumentations are emitted. `libprofile` complains if no bias var is found with continuous mode at runtime.

https://github.com/llvm/llvm-project/commit/d2f77eb8ec620c8068fa471a1d7b9c9756da1913#diff-71a5c7472b215c25675c554f2ee1f3bbc84f29aa47ba2d30543246ad79aeffc0R654-R657

Reply:
I see. Yes, the test is passing by accident on AIX: the dynamic library is not being linked in (because it has no reachable code).